packages/internal/bridge/sdk/src/types/index.ts

```diff
@@ -110,6 +110,15 @@ export enum BridgeMethodsGasLimit { // @TODO test methods on chain and put corre
   MAP_TOKEN_DESTINATION = 200000,
   FINALISE_WITHDRAWAL = 200000,
   APPROVE_TOKEN = 55000,
+
+  DEPOSIT_SOURCE_ETH = 125000,
+  DEPOSIT_SOURCE_USDC = 153000,
+  DEPOSIT_SOURCE_GOG = 137000,
```

Comment:
Maybe we should revise this to 142K gas for GOG, given [this transaction](https://etherscan.io/tx/0x657676ef1c2f53a05e5dd8705ba0835379615fcf1da6abdc87faae98ee67a7fd)